force-app/main/default/classes/BDI_ContactService.cls

```diff
@@ -92,6 +121,26 @@ public with sharing class BDI_ContactService {
         set;
     }
 
+    private Map<String,String> mapDIHomeAddrToContact {
```

Comment:
Can we add a unit test to validate this behaviour?

Reply:
Yeah, I can include it in my existing test method that is creating contact Sobjs using this.